src/MiniProfiler.Shared/Timing.cs

```diff
@@ -77,28 +78,76 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public List<Timing> Children
```

Comment:
Do you think there will be many accesses to the Children property? While collecting timings during the profiler session Children property is no longer used, instead the backing list is used, where the reallocations do not happen. I think the only time when Children property is accessed is during the processing of captured mini profiler snapshot. And even there i wouldn't expect many of the accesses. Maybe I'm missing something, could you give me examples that you think where the property will be accessed a lot?